### DISCUSSION TITLE
Is it possible to set the blog as the default page?

### DISCUSSION THREAD

**Discussion by dhe365** [squidfunk/mkdocs-material#5825] [Q&A] [answered]
Hello everyone! 😊

I was wondering if there's a way to set the blog as the default landing page. It would be awesome to land directly on it when visiting.

Has anyone tried this before or knows if it's feasible? Any guidance would be greatly appreciated!

Thanks in advance and hope you're all having a fantastic day!

**Comment by squidfunk** [comment, marked as answer]
Yes, that's possible. It sounds like you want to build [a standalone blog](https://squidfunk.github.io/mkdocs-material/blog/2022/09/12/blog-support-just-landed/#creating-a-standalone-blog) 😊

**Comment by dhe365** [comment]
Thank you. Sorry I didn't see that. Have a wonderful day!

**Reply by squidfunk** [reply]
No worries, we're currently restructuring our documentation, so that it will be easier to discover what you can do ☺️
